gcloud-java-compute/src/main/java/com/google/gcloud/spi/ComputeRpc.java

```diff
@@ -293,9 +294,44 @@ public Y y() {
   /**
    * Deletes the requested region address.
    *
-   * @return a region operation if request was issued correctly, {@code null} if the address was not
-   *     found
+   * @return a region operation if the request was issued correctly, {@code null} if the address was
+   *     not found
    * @throws ComputeException upon failure or if region is not found
    */
   Operation deleteRegionAddress(String region, String address, Map<Option, ?> options);
+
+  /**
+   * Creates a snapshot for the specified disk.
+   *
+   * @return a zone operation if the create request was issued correctly, {@code null} if the disk
+   *     was not found
+   * @throws ComputeException upon failure
+   */
+  Operation createSnapshot(String zone, String disk, String snapshot, String description,
```

Comment:
Sounds good. I think that would make the API nicer if that works and in I also didn't think we should do that in this PR. I was trying to get your thoughts as that seems nice to me.
Opening an issue for it is the right thing to do.